samples/README.md

```diff
@@ -107,6 +107,7 @@ Entity Framework Core
 * Autonomous Database Sample: Demonstrates Oracle EF Core connecting to Oracle Autonomous Database. <br>
 * Dependency Injection Sample: Demonstrates using dependency injection and ASP.NET Core with Oracle EF Core. <br>
 * Getting Started Sample: Demonstrates a basic Oracle EF Core scenario using migrations and scaffolding. <br>
+* JSON Columns Sample: Demonstrates how to create an owned entity, insert, queriy, updates and delete JSON column data. <br>

```

Comment:
insert, queriy, updates -> insert, query, update

Reply:
Thanks for the review! I saw this error yesterday and fixed it already.